assets/js/press-sync.js

```diff
@@ -78,26 +67,40 @@ window.PressSync = ( function( window, document, $ ) {
 		if ( request_time ) {
 			// Estimate time remaining.
 			var remaining_time = ( ( ( total_objects - total_objects_processed ) / 5 ) * request_time ) / 60 / 60;
-				var time_left_suffix = 'hours';
+			var time_left_suffix = 'hours';
 
-				if ( 1 > remaining_time ) {
-					remaining_time = remaining_time * 60;
-					time_left_suffix = 'minutes';
-				}
+			// Shift to minutes.
+			if ( 1 > remaining_time ) {
+				remaining_time = remaining_time * 60;
+				time_left_suffix = 'minutes';
+			}
 
-				// Round to two decimal places, mostly.
-				remaining_time = Math.round( remaining_time * 100 ) / 100;
+			// Round to two decimal places, mostly.
+			remaining_time = Math.round( remaining_time * 100 ) / 100;
 
-				progress_string += ' (' + [ 'Estimated time remaining:', remaining_time, time_left_suffix ].join(' ') + ')';
-			}
+			progress_string += ' (' + [ 'Estimated time remaining:', remaining_time, time_left_suffix ].join(' ') + ')';
```

Comment:
We should switch these out of l10n strings from the plugin at some point (along with 'hours' and 'minutes'). The fact that this estimate exists is :100:

Reply:
I definitely agree that we should move these to properly localized strings in the future.